dist/dialog.linker.build.js

```diff
@@ -24,10 +24,19 @@ window.jQuery = window.jQuery || window.shoestring;
 		this.$background = !this.$el.is( '[data-nobg]' ) ?
 			$( doc.createElement('div') ).addClass( cl.bkgd ).appendTo( "body") :
 			$( [] );
+
+		// when dialog first inits, save a reference to the initial hash so we can know whether
+		// there's room in the history stack to go back or not when closing
+		this.initialLocationHash = w.location.hash;
+
+		// the dialog's url hash is different from the dialog's actual ID attribute
+		// this is because pairing the ID directly makes the browser jump to the top of the dialog,
+		// rather than allowing us to space it off the top of the viewport.
+		// also, if the dialog has a data-nohistory attr, this property will prevent its findability for onload and hashchanges
+		this.nohistory = this.$el.is( '[data-dialog-nohistory]' );
```

Comment:
Nit: I think the convention we (and others) normally is: `is<adjective>` or `has<noun>`. So in this case it would be `this.hasHistory`.